src/map/battle.cpp

```diff
@@ -5630,8 +5621,6 @@ static struct Damage battle_calc_weapon_attack(struct block_list *src, struct bl
 		case CR_ACIDDEMONSTRATION:
 		case GN_FIRE_EXPANSION_ACID:
 #endif
-		case SO_VARETYR_SPEAR:
-			return wd; //These skills will do a GVG fix later
```

Comment:
Is this `return wd;` line supposed to be removed aswell?
Wouldn't it affect the other skills above it?